packages/mobile/src/components/details-tile/DetailsTileActionButtons.tsx

```diff
@@ -105,11 +118,35 @@ export const DetailsTileActionButtons = ({
     />
   )
 
+  const editButton = (
+    <IconButton
+      fill={neutralLight4}
+      icon={IconPencil}
+      onPress={onPressEdit}
+      styles={{ icon: styles.actionButton }}
+    />
+  )
+
+  const publishButton = (
+    <IconButton
+      fill={neutralLight4}
+      icon={IconRocket}
+      // TODO: Add isDisabled based on if playlist is publishable logic
```

Comment:
just checking this is a todo for not this pr?

Reply:
Yup, for later when the hidden/private logic is added